pkgs/development/python-modules/trustme/default.nix

```diff
@@ -32,8 +33,12 @@ buildPythonPackage rec {
     futures
   ];
 
+  # aarch64-darwin forbids W+X memory, but this tests depends on it:
+  # * https://github.com/pyca/pyopenssl/issues/873
+  pytestFlags = lib.optionalString (stdenv.isDarwin && stdenv.isAarch64) "-k 'not test_pyopenssl_end_to_end'";
```

Comment:
This is a bit unrelated but can we convert this into pytestCheckHook and set disabledTests instead?

Reply:
Sure, amended